src/RA_Integration.vcxproj

```diff
@@ -353,11 +353,14 @@
       <MultiProcessorCompilation>true</MultiProcessorCompilation>
       <DisableSpecificWarnings>4091</DisableSpecificWarnings>
       <ConformanceMode>true</ConformanceMode>
+      <LinkTimeCodeGeneration>UseFastLinkTimeCodeGeneration</LinkTimeCodeGeneration>
```

Comment:
During release mode, the linker keeps on turning this on. So I just turned it on explicitly.